src/lib/server.js

```diff
@@ -15,10 +15,24 @@ const app = http.createServer((request, response) => {
       if (parsedRequest.method === 'GET' && parsedRequest.url === '/') {
```

Comment:
i can't tag it on this review bc you didn't change the specific line im looking for.. but your cow is meant to talk! i read that you were having trouble installing cowsay, but its working properly.   you do need to have a look at the functionality under your if statement with the post request method and change the code a bit to get your cow to talk properly.  otherwise great job on this lab!